src/platform/android/org_easyrpg_player_player_EasyRpgPlayerActivity.cpp

```diff
@@ -2,41 +2,37 @@
  * This file is part of EasyRPG Player
  *
  * Copyright (c) 2017 EasyRPG Project. All rights reserved.
```

Comment:
At least this header still has the © year, a good chance to remove it

Reply:
and not touched since 6 years xD